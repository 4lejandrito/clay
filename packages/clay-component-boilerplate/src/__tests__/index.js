import React from 'react';
import TestRenderer from 'react-test-renderer';
import ClayComponentBoilerplate from '../index';

describe('ClayComponentBoilerplate', function() {
	it('should render', () => {
		const testRenderer = TestRenderer.create(<ClayComponentBoilerplate />);

		expect(testRenderer.toJSON()).toMatchSnapshot();
	});
});
