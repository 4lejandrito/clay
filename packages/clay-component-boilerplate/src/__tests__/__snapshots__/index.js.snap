// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClayComponentBoilerplate should render 1`] = `
<div>
  Clay Component Boilerplate!
</div>
`;
